elements/rh-footer/rh-footer-lightdom.css

```
@@ -20,3 +20,21 @@ rh-footer :is([slot^=links], [slot=footer-links-primary], [slot=footer-links-sec
 rh-footer :is([slot^=links], [slot=footer-links-primary], [slot=footer-links-secondary]) a:is(:hover, :focus-within) {
   text-decoration: underline;
 }
+
+/* Via pfe-base.css */
+rh-footer a {
+  color: var(--pfe-broadcasted--link, #06c);
```

these should be `--rh-`, it's our party :)

```suggestion
  color: var(--rh-color-link-inline-on-light, var(--rh-color-blue-500, #06c));
```